fili-core/src/test/java/com/yahoo/bard/webservice/models/druid/client/impl/TestDruidWebService.java

```diff
@@ -90,7 +90,7 @@ public TestDruidWebService() {
     /**
      * If TestDruidWebService#throwable is set, invokes the failure callback.  Otherwise invokes success or failure
      * dependent on whether TestDruidWebService#statusCode equals 200.
-     * Note that since this doesn't send requests to druid all the responses will be null or {@link FailedFuture}.
+     * Note that since this doesn't send requests to druid all the responses will be null or {@link CompletedFuture}.
```

Comment:
The javadoc plugin is able to correctly recognize this and inject the html link in other classes I've tried (like in CompletedFuture). Did you generate the javadoc and it didn't work correctly?